main.go

```diff
@@ -2,16 +2,15 @@
 // the current directory or a specified directory, on the next free
 // ephemeral port or a specified port.
 //
-//     go get foxygo.at/servedir
-//     servedir --help
-//     usage: servedir [-a] [-p <port>] [<dir>]
+//	go run foxygo.at/servedir@latest --help
+//	usage: servedir [-a] [-p <port>] [<dir>]
```

Comment:
seeing this makes me want to have `gofmt` or `gofumpt` as part of the build process, not as part of this pr. just noting.

Arguably the reformat and help wording improvement could be part of a different commit?

Reply:
Since I was re-doing the `go get` to `go run`, it seemed fine to do the other reformatting in this commit. Furthermore since the flags are re-worked in this commit, reworking the comments with the flags seems to fit to me too. It's all flags :flags: \</wave-hand>